### ISSUE TITLE
Browser down when bot sleeping

### ISSUE THREAD

**Issue by Archon-** [social-manager-tools/twitter-bot-lib#13]
### Problem
Bot for the most of the time (depends on confioguration activity but still) sleeps but Puppeteer keeps browser open even then. It reserves unnecessary machine resources.

### Solution
Would be nice to have browser turned off when bot sleeping. Before closing browser we could dump user session and restore it back when waking up.

### Benefits
More machine resources saved.

I'm happy to prepare some implementation of this. Please, let me know what do you thing. Would it be useful?

**Comment by ptkdev** [comment]
This is hard because restore browser with same session is complicated. Puppeeter use little machine resources: 1 tab of chrome. At moment this not is priority.

**Comment by Archon-** [comment]
That's true. Puppeteer is pretty economical when it comes to resources. However I'd like to optimize it anyway. It's quite important specially in a way I'm using it. Anyway, I'll play around that and check restoring session.

**Comment by ptkdev** [comment]
@Archon- If you create stable release with dump user session send pull request :D

**Comment by Archon-** [comment]
@ptkdev  Here you go. #15
